build.sbt

```diff
@@ -78,4 +78,14 @@ lazy val scoverageExcludePatterns = List(
       resolvers += Resolver.bintrayRepo("hmrc", "releases"),
       resolvers += Resolver.jcenterRepo
     )
+    .settings(
+      scalacOptions ++= List(
+        "-Yrangepos",
+        "-Xlint:-missing-interpolator,_",
+        "-Yno-adapted-args",
+        "-feature",
+        "-unchecked",
+        "-language:implicitConversions",
+        "-P:silencer:pathFilters=views;routes;TestStorage"
```

Comment:
I think we don't have views in the backend so we could take that out although it doesn't do any harm either ways